client/src/App.js

```diff
@@ -20,6 +24,26 @@ export const PATHS = {
   register: "/register",
 }
 
+/*
```

Comment:
Each component that needs to access/write to store, needs these mapState methods defined. These methods are passed in as arguments to `connect`